packages/nimble-components/src/menu-button/tests/menu-button.stories.ts

```diff
@@ -5,16 +5,25 @@ import {
     createUserSelectedThemeStory,
```

Comment:
The button mdx has a big section about how to use the appearance modes and appearance variants but we don't include that section for the menu button, toggle button, or anchor button even though they now (or in some cases already did) support the same configuration.

Do you know if the guidance is the same for all types of buttons or if there are some states that are not recommended for some button types? You may need to consult with Brandon about this. If the guidance is different then we should document it in each component page. If it's the same then we should add something to each page. I'd be fine with it being a link to the button docs. The other alternative would be to find a way to share it but swap out the components used, which may not be worth the effort.

Reply:
For now, I've gone ahead and added that styling section to each of the other button docs. @NIbokeefe is it reasonable to show the same guidance about appearances and appearance variants for anchor, menu, and toggle buttons?

I made some minor edits that came up while doing this. E.g. removed unhelpful `Styling / Theme` section from button docs, made a couple minor wording edits to the button appearance section, and improved wording of spinner's `Styling / Theme` section.

Reply:
@m-akinc 
CC: @jattasNI 

Yes, we can use this solution consistently for all buttons, Action, Toggle, Anchor and Menu.  I've updated the buttons on my end in Figma to reflect those updates.